app/pb_kits/playbook/pb_hashtag/_hashtag.jsx

```diff
@@ -14,30 +17,28 @@ type HashtagProps = {
   url?: String,
 }
 
-const HashType = {
-  "home": "H#",
-  "project": "P#",
-  "default": "#"
+const typeMap = {
+  'home': 'H#',
+  'project': 'P#',
+  'default': '#',
 }
 
 const Hashtag = ({
   className,
-  data,
   dark = false,
-  id,
   text,
   type,
-  url
-} : HashtagProps) => {
-  const darkClass = dark === true ? "dark" : ""
-
-  return (
-    <div className={`pb_hashtag_kit_${darkClass}`}>
-      <a href={url}>
-        <Badge variant="primary" text={HashType[type] + text} dark={dark}></Badge>
-      </a>
-    </div>
-  )
-}
+  url,
+}: HashtagProps) => (
+  <span className={classnames(className, buildCss('pb_hashtag_kit', { 'dark': dark }))}>
```

Comment:
I believe you should be able to omit the initial `classnames` here:

```jsx
 <span className={buildCss('pb_hashtag_kit', className, { 'dark': dark })}>
```

Reply:
Everything passed to `buildCss` will be joined with an underscore, so that wouldn't produce the same result.

Reply:
yeah, nvm, I realized that after the fact 👍